sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosItemRequestOptions.java

```diff
@@ -100,18 +100,25 @@ public CosmosItemRequestOptions setIfNoneMatchETag(String ifNoneMatchETag) {
      *
      * @return the consistency level.
      */
-
-    ConsistencyLevel getConsistencyLevel() {
+    public ConsistencyLevel getConsistencyLevel() {
         return consistencyLevel;
     }
 
     /**
-     * Sets the consistency level required for the request.
+     * Sets the consistency level required for the request. The effective consistency level
+     * can only be reduce for read/query requests. So when the Account's default consistency level
+     * is for example Session you can specify on a request-by-request level for individual requests
+     * that Eventual consistency is sufficient - which could reduce the latency and RU charges for this
+     * request but will not guarantee session consistency (read-your-own-write) anymore
+     * NOTE: If the consistency-level set on a request level here is higher than the default consistency
+     * level set when constructing the CosmosClient instance via CosmosClientBuilder.consistencyLevel
+     * session token capturing also needs to be enabled by calling
+     * CosmosClientBuilder:sessionCapturingOverrideEnabled(true).
```

Comment:
`CosmosClientBuilder:sessionCapturingOverrideEnabled(true)` would only be needed when you set the request consistency to Session.

This is not needed if you set the request consistency to Strong nor for Bounded-Staleness.